package.json

```diff
@@ -25,6 +25,7 @@
     "husky": "^7.0.0",
     "jest": "^27.5.1",
     "jest-extended": "^2.0.0",
+    "jest-sorted": "^1.0.14",
```

Comment:
I would like you to move any jest packages further down into your devDependencies object. This section is only for packages that are needed for the app to run when in production. As we wont be testing in production, we dont want these packages to be part of the final build.